health/health.go

```diff
@@ -0,0 +1,30 @@
+// health is depended on the code generated by protobuf
+// For the users who want to implement their own stuff,
+// should rewrite this service.
+package health
+
+import (
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	healthpb "google.golang.org/grpc/health/grpc_health"
+)
+
+func HealthCheck(t time.Duration, cc *grpc.ClientConn) error {
+	ctx, _ := context.WithTimeout(context.Background(), t)
+	hc := healthpb.NewHealthCheckClient(cc)
+	req :=new(healthpb.HealthCheckRequest)
+	if _,err := hc.Check(ctx, req); err != nil {
```

Comment:
It seems you do not care the value of "err". You can do

_, err := hc.Check(ctx, req)
return err